assets/scss/components/stencil/productView/_productView.scss

```diff
@@ -72,7 +90,7 @@
         position: absolute;
         right: 0;
         top: 0;
-        width: auto;
+        width: 100%;
```

Comment:
won't this stretch the image to the width of the outside container. This could cause image to stretch when they are loaded with a small dimension like 10x10

Reply:
Oh, I remembered why I did this. There's currently an issue when the browser width is extremely small, the thumbnails will extend outside their borders, like so:

![screen shot 2017-10-20 at 11 10 39 am](https://user-images.githubusercontent.com/32500994/31835725-8dd658b8-b587-11e7-925c-3c8d8ec52d4b.png)

I would have liked to use max-width, but unfortunately we're already setting max-width at 50px here. I don't know of a CSS trick to essentially say "max of 50px or 100%, whichever is less", but perhaps there is one.

In any event, this doesn't really have to be a part of this ticket, so I changed it back.